titus-server-master/src/main/java/com/netflix/titus/master/mesos/kubeapiserver/KubeConstants.java

```diff
@@ -41,21 +41,33 @@ private KubeConstants() {
 
     public static final String NODE_LABEL_INSTANCE_TYPE = "beta.kubernetes.io/instance-type";
 
+    /**
+     * Common prefix for Titus node annotations/labels and taints.
+     */
+    public static final String TITUS_NODE_DOMAIN = "node.titus.netflix.com/";
+
+    /**
+     * Common prefix for Titus V3 job API specific annotations/labels.
+     */
+    public static final String TITUS_V3_JOB_DOMAIN = "v3.job.titus.netflix.com/";
+
     /*
-     * Titus node labels.
+     * Titus pod labels.
      */
 
-    public static final String NODE_LABEL_MACHINE_ID = "titus/machine-id";
+    public static final String POD_LABEL_JOB_ID = TITUS_V3_JOB_DOMAIN + "jobId";
```

Comment:
I think we should stick with DNS naming conventions for annotations and labels, so all lowercase characters:

https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names
https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set

I suggest `.../job-id` and `.../task-id`. See how `machine-id` and `machine-group` were chosen below, rather than `machineId`. Alternatively, it could also be `.../jobid` and `.../taskid`.

Reply:
I like the idea. Changing this to `job-id` and `task-id`.